teamscale-maven-plugin/src/main/java/com/teamscale/tia/maven/TiaCoverageConvertMojo.java

```diff
@@ -0,0 +1,135 @@
+package com.teamscale.tia.maven;
+
+import com.google.common.base.Strings;
+import com.teamscale.jacoco.agent.options.AgentOptionParseException;
+import com.teamscale.jacoco.agent.options.ClasspathUtils;
+import com.teamscale.jacoco.agent.options.FilePatternResolver;
+import com.teamscale.jacoco.agent.util.Benchmark;
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+import shadow.com.teamscale.client.TestDetails;
+import shadow.com.teamscale.report.EDuplicateClassFileBehavior;
+import shadow.com.teamscale.report.ReportUtils;
+import shadow.com.teamscale.report.testwise.ETestArtifactFormat;
+import shadow.com.teamscale.report.testwise.TestwiseCoverageReportWriter;
+import shadow.com.teamscale.report.testwise.jacoco.JaCoCoTestwiseReportGenerator;
+import shadow.com.teamscale.report.testwise.model.TestExecution;
+import shadow.com.teamscale.report.testwise.model.factory.TestInfoFactory;
+import shadow.com.teamscale.report.util.ClasspathWildcardIncludeFilter;
+import shadow.com.teamscale.report.util.CommandLineLogger;
+import shadow.com.teamscale.report.util.ILogger;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Batch converts all created .exec file reports into a testwise coverage report.
+ */
+@Mojo(name = "testwise-coverage-converter", defaultPhase = LifecyclePhase.VERIFY,
```

Comment:
`defaultPhase = LifecyclePhase.VERIFY` means we don't do the conversion when executing `mvnw test` but only when calling `mvn verify`. Can we make it work for both out of the box?

Reply:
This is intentional, as this makes sure that the plugin is executed after both the unit and the integration tests are run. Setting the default to `LifecyclePhase.TEST` would mean that the coverage is converted before the integration tests are executed.

If one desires to execute the plugin after running only mvnw test, you have to either explicitely execute the goal in the run command something like ->` mvnw test com.teamscale.tia.maven:testwise-coverage-converter`

Or by changing the `pom.xml` to execute the mojo at a specific phase like this:
```
                    <execution>
                        <id>test-report-conversion</id>                    
                        <goals>
                            <goal>testwise-coverage-converter</goal>
                        </goals>
                        <phase>test</phase>
                    </execution>
```

Reply:
I see, thanks for the explanation.